src/lib/rest/rest.cpp

```diff
@@ -265,6 +265,15 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
 }
 
 
+/* ****************************************************************************
+*
+* uriParamGet -
+*/
+int uriParamGet(ConnectionInfo *ciP, const char* ckey, const char* val)
+{
+  return uriArgumentGet((void *) ciP, (MHD_ValueKind)0, ckey, val);
+}
+
```

Comment:
The name of this function is a little funny ...
The original name, **uriArgumentGet** came from 'getting uri param from MHD', but this function is more like 'pushing param to ConnectionInfo', right?

Perhaps we should rename **uriArgumentGet** to **uriArgumentReceive** and **uriParamGet** to **uriParamPush** ?